src/Plugins/ComplexCore/CMakeLists.txt

```diff
@@ -104,7 +104,7 @@ set(AlgorithmList
   PointSampleTriangleGeometry
   QuickSurfaceMesh
   RawBinaryReader
-  ResampleImageGeom
```

Comment:
Was ResampleImageGeom meant to be removed?

Reply:
no